src/app/config/mint.ts

```diff
@@ -9,6 +9,10 @@ const supportedMints: Record<string, MintInfo> = {
     address: '4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU', // devnet
     decimals: 6,
   },
+  'USDT': {
+    address: '4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU', // lol what's the devnet USDT address?? can't find it anywhere.
+    decimals: 6,
+  },
```

Comment:
Let's use PYUSD!
```suggestion
  'PYUSD': {
    address: 'CXk2AMBfi3TwaEL2468s6zP8xq9NxTXjp9gjMgzeUynM',
    decimals: 6,
  },
```

Reply:
agreed let's do it in a follow up when we enable token 2022